packages/webapp/src/components/HoverPill/index.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { Tooltip } from '@mui/material';
+import styles from './styles.module.scss';
+import { Semibold, Main } from '../Typography';
+
+export interface HoverPillProps {
+  items: string[];
+}
+
+export const HoverPill = ({ items }: HoverPillProps) => {
+  const { t } = useTranslation();
+
+  const hoverContent = (
+    <>
+      {items.map((item, index) => (
+        <Main key={index} className={styles.detailText}>
+          {item}
+        </Main>
+      ))}
+    </>
+  );
+
+  // https://mui.com/material-ui/react-tooltip/#distance-from-anchor
+  const PopperProps = {
+    modifiers: [
+      {
+        name: 'offset',
+        options: {
+          offset: [0, -8],
```

Comment:
There's something super small that's bothering me 😅 which I think is actually buggy behavior from MUI, but hopefully we can fix it? When you hover over the pill and the tooltip opens, if you then hover back to the pill there's a space within it where hovering doesn't work and the styling turns back to the "unhovered" style. I recorded a video which hopefully shows it better.

https://github.com/LiteFarmOrg/LiteFarm/assets/20230771/357fb51f-ff6b-494a-b683-929fb2d3061f

I couldn't figure out why this was happening, but it turns out that the tooltip has a bottom margin and that bit of it is overlapped with the pill.

<img width="1792" alt="Captura de pantalla 2024-02-08 a la(s) 10 05 28" src="https://github.com/LiteFarmOrg/LiteFarm/assets/20230771/c8ca6d72-2c48-41af-aefa-4260fc27ec36">


The solution seems to be to apply a `margin: 0` style to the tooltip to override MUI's margin, and then adjust the offset to be a positive number so that there's still some space between the tooltip and the pill. The only problem is that the MUI selector is more specific than the `.hoverDetails` one, so for it to work the custom selector needs to be more specific. One way that I found to do this without using `!important` is to apply the `hoverDetails` styling to MUI's styling for specific tooltip placements, like this:

```
<Tooltip
      classes={{
        tooltip: styles.hoverDetails,
        tooltipPlacementTop: styles.hoverDetails,
      }}
      ...
>

```

Reply:
Ah, so that's why the offset had to be negative in the first place! These default component styles always get me!! 🤣 

I couldn't get it to work just like you described though (with override classes but without `!important`); could you check again? 😅 And once the overall offset is changed to positive I think we'll need to override classes for more placements (I would think at least `tooltipPlacementBottom`)? If so I think it might be okay (and at least more succinct) to go the `!important!` route...

Reply:
Ah no, you're right, it doesn't work, I had probably done a different change and thought it worked when it didn't. At some point I just repeated the same selector to increase the specificity, like `.hoverDetails.hoverDetails.hoverDetails` but I think anyone reading that would wonder what's going on there probably 😅 so I'm alright with keeping the important. For the record this is one of the things I strongly dislike about MUI, which is how hard it is to override their styling